docs/kss/public/less/_common.css

```diff
@@ -0,0 +1,112 @@
+body.kss-body {
+  background: #ffffff;
+  overflow: visible;
+  font-size: 15px;
+  font-family: 'Open Sans', sans-serif;
+  padding: 0;
+  max-width: 100%;
+  padding-top: 100px;
+  padding-right: 0!important;
+}
+#kss-wrapper {
+  height: auto;
+}
+.active.demo-container {
+  background-color: #e0e9f0;
+  padding: 15px;
+}
+#kss-header h1 {
+  font-size: 1.5em;
+  margin-top: 1.66666667em;
+  margin-left: 1.33333333em;
+}
+h1.kss-title {
+  color: #369ad6;
+  margin-bottom: 30px;
+  font-weight: 500;
+  font-family: 'Dosis', sans-serif;
+}
+#kss-main {
+  padding-left: 15px;
+  padding-bottom: 40px;
+}
+#kss-main.col-md-12 {
+  max-width: 1000px;
```

Comment:
I'm not quite sure about this one, it doesn't seem to make sense as there should be a container  that sets the max width.